src/app/lib/button/button.component.ts

```diff
@@ -1,16 +1,42 @@
-import { Component, EventEmitter, Output, Input } from '@angular/core';
-import { ButtonOptions } from './button-options';
+/* tslint:disable:component-selector */
```

Comment:
At some point we'll want to establish our tslint rules and perhaps have an [.editor config file](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) that we check in with the library pull requests don't have as much noise from formatting.

Reply:
I totally agree and didn't realize until @waldnercharles had said that I'm using spaces instead of tabs and I'm using 2 spaces per tab. I created a new issue #8 that is intended to fix this issue. I'm going to hold off on getting this pull request merged in until that is resolved, since we're in no hurry